peachjam/adapters/adapters.py

```diff
@@ -74,30 +75,24 @@ def update_document(self, url):
             LegalInstrument,
             Legislation,
             Locality,
-            Work,
         )
 
         logger.info(f"Updating document ... {url}")
 
         document = self.client_get(f"{url}.json").json()
-        frbr_uri = document["frbr_uri"]
```

Comment:
keep this, and parse it with `FrbrUri.parse` which will then give you all the constituent fields, including `author`, which we don't expose in the Indigo API.